index.bs

```diff
@@ -2161,25 +2169,25 @@ This enumeration's values describe the [=[RP]=]'s requirements for [=resident cr
 
 <div dfn-type="enum-value" dfn-for="ResidentKeyRequirement">
     :   <dfn>forbidden</dfn>
-    ::  This value indicates the authenticator MUST only create a non-resident credential.
+    ::  This value indicates the authenticator MUST create a [=Non-Resident Credential=].
 
     :   <dfn>indifferent</dfn>
-    ::  This value indicates the client may use any authenticator. The authenticator MAY create a resident or non-resident credential at 
-        its discretion.
+    ::  This value indicates the client MAY use any authenticator. The authenticator MAY create a [=Resident Credential=] or 
+        [=Non-Resident Credential=] at its discretion.
 
     :   <dfn>preferred</dfn>
-    ::  This value indicates the client should try to use an authenticator capable of [=resident credentials=] in preference to one that
-        is incapable of [=resident credentials=]. If no authenticator is capable of [=resident credentials=], the client may use any
-        authenticator capable of non-resident credentials.
+    ::  This value indicates the client SHOULD try to use an authenticator capable of a [=resident credentials|Resident Credential=] 
```

Comment:
You don't need to specify `[=resident credentials|Resident Credential=]`. Bikeshed automatically equates simple -s plural forms and capitalizations, so this can just be `[=resident credential=]`.

Reply:
Thanks - done.